dix/req.c

```diff
@@ -1365,6 +1366,8 @@ static void dix__rop(struct m0_dix_req *req, const struct m0_bufvec *keys,
 
 	keys_nr = keys->ov_vec.v_nr;
 	M0_PRE(keys_nr != 0);
+	/* We support only one KV pair per request in DTM0. */
+	M0_PRE(ergo(req->dr_dtx != NULL, keys_nr == 1));
```

Comment:
Question:
If the KV pair is more than 1, we will not enable DTM0 for this request?
Or when DTM0 is enabled, we only support single KV pair  request?

Reply:
When DTM0 is enabled, we only support single KV pair request?